IntegrationTests/allocation-counter-tests-framework/run-allocation-counter.sh

```diff
@@ -176,9 +182,10 @@ function build_package() {
 EOF
         ln -s "$f" "Sources/Test_$module/file.swift"
         ln -s "../../scaffolding.swift" "Sources/Test_$module/"
-        if [[ -n "$shared_file" ]]; then
-            ln -s "$shared_file" "Sources/Test_$module/shared.swift"
-        fi
+        for shared_file in "${shared_files[@]}"; do
```

Comment:
I think this will break if there are _no_ shared files. Or did you test that?

```suggestion
        for shared_file in "${shared_files[@]+"${shared_files[@]}"}"; do
```

Reply:
Totally right it was busted if there were no shared files, thanks!